packages/base/src/3dview/helpers.ts

```diff
@@ -167,20 +168,20 @@ export function buildShape(options: {
   // it's too bad Three.js does not easily allow setting uniforms independently per-mesh
   const material = new THREE.MeshPhongMaterial({
     color,
-    side: THREE.DoubleSide,
     wireframe: false,
     flatShading: false,
     clippingPlanes,
     shininess: 0
   });
 
   const geometry = new THREE.BufferGeometry();
+
   geometry.setIndex(triangles);
   geometry.setAttribute(
     'position',
     new THREE.Float32BufferAttribute(vertices, 3)
   );
-  geometry.setAttribute('normal', new THREE.Float32BufferAttribute(normals, 3));
+  geometry.computeVertexNormals();
```

Comment:
Is this really needed? If nodes are ordered correctly, we can save a tiny bit of computation and let the GPU figure out the normals when rendering

Reply:
Let's not block the PR for this, it's a detail